package.json

```diff
@@ -1,18 +1,9 @@
 {
   "name": "currency-cloud",
   "description": "Currencycloud API v2 JavaScript client",
-  "version": "4.5.0",
+  "version": "4.5.1",
```

Comment:
@alexbor it is not necessary to have a new release, there is no functional or dependency change just test updates.